### PR TITLE
Test coverage

### PR DESCRIPTION
**Pull request by devalirizwan**

### CODE REVIEW

**Review by brentkearney — CHANGES_REQUESTED** [review]
Please fix issues identified by Codacy

File: .rubocop.yml [hunk 1]
```diff
@@ -27,5 +27,14 @@ Style/BlockComments:
...
+  Exclude:
+    - 'spec/**/*.rb'
+Metrics/ClassLength:
+  Enabled: false
+Metrics/ModuleLength:
+  Enabled: false
+Lint/EmptyBlock:
+  Enabled: false  
+Naming/VariableNumber:
+  Enabled: false
```
Comment (brentkearney) [inline .rubocop.yml:40]: Skipping the BlockLength checks in the specs is OK. Please re-enable the other tests, and refactor methods and modules that are too long, etc..

File: app/controllers/locations_controller.rb [hunk 2]
```diff
@@ -27,20 +27,21 @@ def create
...
         format.html do
-          render :edit, status: :unprocessable_entity,
-                        error: "Unable to update location."
+          render :edit, status: :unprocessable_entity, error: "Unable to update location."
         end
         format.json { render json: @location.errors, status: :unprocessable_entity }
       end
     end
   end
+  # rubocop:enable Metrics/MethodLength
```
Comment (brentkearney) [inline app/controllers/locations_controller.rb:44]: Please remove these # rubocop comments. I will skip it in Codacy if it fails.

File: app/controllers/proposal_fields_controller.rb [hunk 3]
```diff
@@ -27,6 +31,7 @@ def create
                   alert: @proposal_form.errors
     end
   end
+  # rubocop:enable Metrics/MethodLength
```
Comment (brentkearney) [inline app/controllers/proposal_fields_controller.rb:34]: Please remove the skip

**Comment by brentkearney** [comment]
Please remove all  # rubocop:enable Metrics/MethodLength

**Comment by devhassanmahmood** [comment]
Hey - @brentkearney Can you please look why tests are running for subjects_controller.rb ? I have removed subjects_controller file and also removed from spec/requests folder. Why it is considering it?

**Pull request closed without merging**